### PR TITLE
fixed rowRenderer def to pass style and key

### PR DESCRIPTION
**Pull request by kikemx78**
Please fill in this template.

- [ ] Make your PR against the `master` branch.
- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `tsc` without errors.
- [ ] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by andrewdewaal — APPROVED** [review]

File: react-virtualized/index.d.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ declare module "react-virtualized" {
         rowCount: number;
         rowGetter?: (info: { index: number }) => any;
         rowHeight: number | ((info: { index: number }) => number);
-        rowRenderer?: (info: { index: number, isScrolling: boolean }) => React.ReactNode;
+        rowRenderer?: (info: { index: number, isScrolling?: boolean, key: any, style: string }) => React.ReactNode;
```
Comment (ryprice) [inline react-virtualized/index.d.ts]: I actually made the same change so I can just delete my PR, but I'm not 100% sure key exists on Table rowRenderer, just based on the docs. https://github.com/bvaughn/react-virtualized/blob/master/docs/Table.md

File: react-virtualized/index.d.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ declare module "react-virtualized" {
         onScroll?: (info: { clientHeight: number, scrollHeight: number, scrollTop: number }) => void;
         overscanRowCount?: number;
         rowHeight: number | ((info: { index: number }) => number);
-        rowRenderer: (info: { index: number, isScrolling: boolean }) => React.ReactNode;
+        rowRenderer: (info: { index: number, isScrolling?: boolean, key: any, style: string }) => React.ReactNode;
```
Comment (ryprice) [inline react-virtualized/index.d.ts]: `key: string, style: Object`

**Comment by minestarks** [comment]
@kikemx78 Could you please respond to the requested changes? Thank you.

**Comment by kikemx78** [comment]
@minestarks sorry, haven't looked at it. Seems Ok to me

**Comment by minestarks** [comment]
@ryprice since you reviewed this, are you OK with the changes as is?

**Comment by ryprice** [comment]
@minestarks I don't think the style param is the correct type. It should be an `object` according to the documentation so that should really be changed. @kikemx78 does a string work?

The key prop is more of a nit, but we should strive not to use the any type when possible. React's key prop is always `string | number`.

**Comment by kikemx78** [comment]
@ryprice style param should be an object...I totally agree with you..My bad

**Comment by andy-ms** [comment]
@kikemx78 Sorry, something went wrong with merging this. I have to revert it (#15125). Could you re-do this PR?

**Pull request merged**